packages/circuit-ui/components/SelectorGroup/SelectorGroup.stories.tsx

```diff
@@ -48,7 +49,7 @@ export const Base = (args: SelectorGroupProps) => {
 
 Base.args = baseArgs;
 
-export const MultipleSelection = (args: SelectorGroupProps) => {
+export const Multiple = (args: SelectorGroupProps) => {
```

Comment:
Renamed back to `Multiple`, otherwise this breaks the story embed on SB docs pages (we use a story ID that's generated from the story component name, in this case it's `'selector-selectorgroup-multiple'`)